src/mocks/data.js

```diff
@@ -8,6 +8,19 @@ export const user = {
   email: "j.jdoe@home.org",
 };
 
+export const IVAs = [{
+  id: "TEST00014000001",
+  type: "Phone",
+  value: "+441234567890004",
+  status: "Verified"
```

Comment:
As a side note: We need to check that the status enums are translated properly from the backend to the frontend, and that we use enum types consistently throughout the application. Probably after the backend work has been finished.

(The difficulties arise from the different naming conventions in Python and JavaScript and the different kinds of enum types or string literal types on both sides that could be used.)

Reply:
I tackle this in the PR for 642. So we can resolve it here hopefully?